src/Component/Symfony/Session/Filter/Storage/SessionFilterStorage.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Symfony\Session\Filter\Storage;
```

Comment:
```suggestion
namespace Sylius\Bundle\GridBundle\Storage;
```

Reply:
What do you think about Sylius\Bundle\GridBundle\Filter\Session\Storage instead not to forget about the new namespace when moving to one-package architecture?

Reply:
It's still doesn't match existing notation, so generally I'm against it. As source code reader, it leaves bad impression, when codebase uses multiple structuring techniques. 

It's only a one file, so it shouldn't affect future one-package initiative.